packages/@react-spectrum/s2/package.json

```diff
@@ -146,9 +150,6 @@
     "react-aria-components": "^1.5.0"
   },
   "peerDependencies": {
-    "@testing-library/react": "^15.0.7",
```

Comment:
these are not peer dependencies, not sure when they were added, but they are dev dependencies since downstream users are not required to provide them at all